src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+    private static final String DEFAULT_CIRCLE_COLOR = "White";
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        return createRandomFigure(TypeOfFigure.values()[random
+                .nextInt(TypeOfFigure.values().length)]);
+    }
+
+    public Figure createRandomFigure(TypeOfFigure types) {
+        Figure figure;
+        switch (types) {
+            case CIRCLE :
+                figure = new Circle(colorSupplier.getRandomColor(), random.nextInt());
+                break;
+            case SQUARE:
+                figure = new Square(colorSupplier.getRandomColor(), random.nextInt());
+                break;
+            case RECTANGLE:
+                figure = new Rectangle(colorSupplier.getRandomColor(), random.nextInt(),
+                        random.nextInt());
+                break;
+            case RIGHT_TRIANGLE:
+                figure = new RightTriangle(colorSupplier.getRandomColor(), random.nextInt(),
+                        random.nextInt());
+                break;
+            case ISOSCELES_TRAPEZOID:
+                figure = new IsoscelesTrapezoid(colorSupplier.getRandomColor(), random.nextInt(),
+                        random.nextInt(), random.nextInt());
+                break;
+            default:
+                figure = getDefaultFigure();
```

Comment:
```suggestion
            default:
                figure = new IsoscelesTrapezoid(colorSupplier.getRandomColor(), random.nextInt(),
                        random.nextInt(), random.nextInt());
                break;
```

Reply:
It was in common mistakes

Reply:
I think my code here is right. because readme.md file say next: The other half of figures should have the same, default parameters.For this purpose create a new method in the FigureSupplier class:
public Figure getDefaultFigure() - this method should always return white circle with radius 10. 
Also : https://github.com/mate-academy/jv-oop-advanced/pull/693#discussion_r984499728

Reply:
oh, sorry. I see it. My bad.